### ISSUE TITLE
Uncaught TypeError: ref.is is not a function

### ISSUE THREAD

**Issue by gold** [rjaviervega/tree-view-git-modified#20]
Step to repro:

1. Execute keyboard shortcut to toggle the display of the tree panel throws error.

**Atom**: 1.16.0 x64
**Electron**: 1.3.13
**OS**: Ubuntu 16.04.2
**Thrown From**: [tree-view-git-modified](https://github.com/rjaviervega/tree-view-git-modified) package 0.7.1

### Stack Trace

Uncaught TypeError: ref.is is not a function

```
At /home/gold/.atom/packages/tree-view-git-modified/lib/tree-view-git-modified.coffee:35

TypeError: ref.is is not a function
    at /packages/tree-view-git-modified/lib/tree-view-git-modified.coffee:35:31)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:259:29)
    at /app.asar/src/command-registry.js:3:59
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:599:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:390:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/app.asar/src/window-event-handler.js:106:36)
    at /app.asar/src/window-event-handler.js:3:59)
```

### Commands

```
     -0:55.9.0 intentions:highlight (input.hidden-input)
     -0:55.6.0 tree-view:toggle (input.hidden-input)
  4x -0:18.7.0 advanced-open-file:toggle (input.hidden-input)
```

### Non-Core Packages

```
advanced-open-file 0.16.6 
atom-darcula 0.2.0 
atom-easy-jsdoc 4.12.1 
atom-typescript 11.0.1 
atomic-emacs 0.10.0 
base16-ocean-dark-spacegray 0.13.0 
blame 0.10.3 
busy-signal 1.3.0 
center-line 1.2.4 
chester-atom-syntax 0.1.1 
clipboard-plus 0.5.1 
color-picker 2.2.5 
config-import-export 0.3.1 
cursor-blink-interval 0.3.1 
dark-flat-ui 1.2.4 
darkula-ui 0.1.0 
darkular-syntax 0.3.0 
dracula-theme 2.0.3 
dracula-ui 0.4.1 
duotone-dark-sea-syntax 2.1.0 
editorconfig 2.2.2 
emacs-flow 0.1.2 
emacs-plus 0.11.1 
file-icons 2.1.3 
filesize 2.0.4 
flat-simple-dark-ui 0.1.1 
flat-ui-syntax 0.5.1 
flatland-dark-ui 0.2.5 
fold-functions 0.5.0 
fonts 2.0.0 
git-blame 1.0.1 
git-difftool 0.2.7 
git-plus 7.4.0 
gold-syntax 0.4.0 
goto-definition 1.3.1 
highlight-selected 0.13.1 
idiomatic-dark-syntax 0.1.3 
incremental-search 5.2.2 
intentions 1.1.2 
isearch 0.1.13 
language-lisp 0.2.0 
language-vue 0.21.2 
linter 2.1.2 
linter-eslint 8.1.6 
linter-pylint 2.0.1 
linter-ui-default 1.2.2 
nord-atom-syntax 0.8.0 
nord-atom-ui 0.11.0 
northem-dark-atom-syntax 2.0.0 
northem-dark-atom-ui 2.0.0 
open-on-bitbucket 0.4.0 
open-recent 5.0.0 
photon-syntax 0.9.0 
pigments 0.39.1 
platformio-ide-terminal 2.5.0 
pretty-json 1.6.3 
project-manager 3.3.4 
raml 0.2.0 
react 0.16.2 
spacegray-atom-dark-syntax 1.3.2 
spacegray-dark-ui 0.12.0 
sync-settings 0.8.1 
toggle-gutter 0.2.2 
tree-view-git-modified 0.7.1 
underline-trailing-whitespace 1.4.2 
```

**Comment by Jafnee** [comment]
1. Use keybind for tree-view:toggle

**Atom**: 1.17.0-beta0 x64
**Electron**: 1.3.14
**OS**: Ubuntu 16.10
**Thrown From**: [tree-view-git-modified](https://github.com/rjaviervega/tree-view-git-modified) package 0.7.1


### Stack Trace

Uncaught TypeError: ref.is is not a function

```
At /home/work/.atom/packages/tree-view-git-modified/lib/tree-view-git-modified.coffee:35

TypeError: ref.is is not a function
    at /packages/tree-view-git-modified/lib/tree-view-git-modified.coffee:35:31)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom-beta/resources/app/src/command-registry.js:265:1)
    at /usr/share/atom-beta/resources/app/src/command-registry.js:3:1
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom-beta/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:610:1)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom-beta/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:401:1)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/usr/share/atom-beta/resources/app/src/window-event-handler.js:100:1)
    at HTMLDocument.<anonymous> (/usr/share/atom-beta/resources/app/src/window-event-handler.js:3:1)
```

### Commands

```
     -2:55.3.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-always)
     -2:54.8.0 fuzzy-finder:toggle-buffer-finder (div.tool-panel.tree-view)
     -1:31.1.0 intentions:highlight (input.hidden-input)
     -1:30.7.0 fuzzy-finder:toggle-file-finder (input.hidden-input)
     -1:29.1.0 core:confirm (input.hidden-input)
     -0:58.4.0 intentions:highlight (input.hidden-input)
     -0:57.8.0 pane:split-right-and-copy-active-item (input.hidden-input)
     -0:33.1.0 intentions:highlight (input.hidden-input)
     -0:32.7.0 tree-view:toggle (input.hidden-input)
```

### Non-Core Packages

```
activate-power-mode 1.4.0 
aesthetic-ui 3.0.1 
angular-2-typeScript-snippets 0.6.3 
atom-beautify 0.29.22 
atom-jade 0.3.0 
atom-monokai 0.10.9 
atom-typescript 11.0.2 
auto-detect-indentation 1.3.0 
autocomplete-python 1.8.63 
busy-signal 1.3.0 
change-case 0.6.5 
Chromodynamics 0.0.11 
color-picker 2.2.5 
custom-title 1.0.1 
emmet 2.4.3 
file-icons 2.1.3 
file-watcher 1.1.0 
git-plus 7.5.0 
git-time-machine 1.5.9 
hey-pane 0.2.2 
highlight-selected 0.13.1 
hyperclick 0.0.40 
intentions 1.1.2 
language-ini 1.19.0 
language-markdown 0.21.0 
language-nginx 0.7.2 
line-ending-converter 1.3.2 
linter 2.1.4 
linter-csslint 1.3.4 
linter-eslint 8.1.6 
linter-flake8 2.1.5 
linter-htmlhint 1.3.3 
linter-jshint 3.0.3 
linter-markdown 4.0.0 
linter-php 1.3.2 
linter-pylint 2.0.1 
linter-sass-lint 1.7.4 
linter-tslint 1.1.0 
linter-ui-default 1.2.3 
linter-write-good 0.9.0 
MagicPython 1.0.9 
markdown-preview-plus 2.4.9 
markdown-toc 0.4.2 
minimap 4.27.1 
minimap-autohide 0.10.1 
minimap-cursorline 0.2.0 
minimap-find-and-replace 4.5.2 
minimap-git-diff 4.3.1 
minimap-highlight-selected 4.6.1 
monokai 0.22.0 
nuclide 0.218.0 
pigments 0.39.1 
project-manager 3.3.4 
qolor 0.4.1 
seti-icons 1.4.3 
seti-ui 1.7.0 
sort-lines 0.14.0 
todo-show 1.11.0 
tree-view-git-branch 0.1.2 
tree-view-git-modified 0.7.1 
tree-view-git-status 1.3.1 
wakatime 6.0.14 
zentabs 0.8.8 
```

**Comment by rjaviervega** [comment]
This issues has been fixed on v0.7.2.
